recipes/pyusb/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "pyusb" %}
+{% set version = "1.0.2" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "4e9b72cc4a4205ca64fbf1f3fff39a335512166c151ad103e55c8223ac147362" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  host:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - usb
+    - usb.backend
+
+about:
+  home: http://walac.github.io/pyusb
+  license: BSD
+  license_family: BSD
```

Comment:
add `license_file: LICENSE`

Reply:
Do I Have to explicit create the file ```LICENSE``` with the BAS-3-Clause?